assets/app/scripts/controllers/route.js

```diff
@@ -62,6 +63,11 @@ angular.module('openshiftConsole')
             };
           });
 
+        // List services to display route warnings.
+        DataService.list("services", context, function(services) {
```

Comment:
Dynamic content for the popover doesn't update properly :( #6346

I wanted to switch it to a watch when that is fixed.